test/check-application

```diff
@@ -3207,6 +3207,31 @@ WantedBy=multi-user.target default.target
         self.performContainerAction(IMG_ALPINE, "Start")
         b.wait(lambda: self.getContainerAttr("sys1", "State") == "Running")
 
+    def _testQuadlets(self, *, system: bool = False) -> None:
+        b = self.browser
+        self.login(system=system)
+
+        service_name = "quak"
+        self.createQuadlet(service_name, auth=system)
+        self.waitContainerRow(service_name)
+
+        # Cannot delete a quadlet container, they are managed by .container units
+        b.click(f"#containers-containers tbody tr:contains('{service_name}') .pf-v5-c-menu-toggle")
+        b.wait_visible(self.getContainerAction(service_name, 'Commit'))
+        b.wait_not_present(self.getContainerAction(service_name, 'Delete'))
+        b.click(f"#containers-containers tbody tr:contains('{service_name}') .pf-v5-c-menu-toggle")
+        b.wait_not_present(self.getContainerAction(service_name, 'Commit'))
```

Comment:
This is a wee bit weird/overly specific  -- now the entire .pf-c-menu should go away, right?

Reply:
I strengthened the test.